cmd/honeycloudfront/main.go

```diff
@@ -77,7 +79,33 @@ Your write key is available at https://ui.honeycomb.io/account`)
 				}
 			}
 
-			stater := state.NewFileStater(opt.StateDir, logbucket.AWSCloudFront)
+			var stater state.Stater
+
+			if opt.BackFill < 1 || opt.BackFill > 168 {
+				logrus.WithField("hours", opt.BackFill).Fatal("--backfill requires an hour input between 1 and 168")
+			}
+
+			if opt.HighAvail {
+				svc := dynamodb.New(sess)
+				input := &dynamodb.DescribeTableInput{
+					TableName: aws.String(state.DynamoTableName),
+				}
+				_, err := svc.DescribeTable(input)
+				if err != nil {
+					// For some reason, we cannot write to
+					// the table or access it
+					logrus.WithField("needed", state.DynamoTableName).Fatal("--high-availability requires an existing DynamoDB named tabled appropriately, please refer to the README.")
```

Comment:
As someone reading this log output, I'm not sure what to make of this `needed` key - is that what I need? (no, it's what I have; I need something different.)  Hmm.  Maybe `table_name`? 
Also s/DynamoDB named tabled appropriately/DynamoDB tabled named appropriately/